samples/README.md

```diff
@@ -1,3 +1,15 @@
+# Sample apps for the AWS IoT Device SDK for Embedded C
+
+All samples are written in C unless otherwise mentioned. The these sample apps are included in the SDK and described below.
+ * [`subscribe_publish_sample`](#subscribe-publish-sample) - a simple pub/sub MQTT example
+ * [`subscribe_publish_cpp_sample`](#subscribe-publish-cpp-sample) - a simple pub/sub MQTT example written in C++
+ * [`subscribe_publish_library_sample`](#subscribe-publish-library-sample) - a simple pub/sub MQTT example which builds the SDK as a separate library
+
+ These sample apps are also provided in this SDK.
+ * [`shadow_sample`](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/master/samples/linux/shadow_sample) - a simple device shadow example using a connected window example
+ * [`shadow_sample_console_echo`](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/master/samples/linux/shadow_sample_console_echo) - a sample to work with the AWS IoT Console interactive guide
+ * [`jobs_sample`](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/master/samples/linux/jobs_sample) - Establishes a connection to the AWS IoT MQTT Platform and performs several operations to demonstrate the basic capabilities of the AWS IoT Jobs platform.
```

Comment:
How about the following to be consistent with the above:

```
A sample to demonstrate the capabilities of the AWS IoT Jobs service
```

Please ensure either all the sample descriptions end with period or none does.

Reply:
Edited all descriptions for consistency.